basyx.submodelrepository/basyx.submodelrepository-core/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/SubmodelRepository.java

```diff
@@ -79,7 +80,7 @@ public interface SubmodelRepository {
 	 * @param submodel
 	 * @throws CollidingIdentifierException
 	 */
-	public void createSubmodel(Submodel submodel) throws CollidingIdentifierException;
+	public void createSubmodel(Submodel submodel) throws CollidingIdentifierException,MissingIdentifierException;
```

Comment:
Please use BaSyx formatter which is available [here](https://wiki.eclipse.org/File:BaSyx_Formatting.zip).

Reply:
done